recipes/slipcover/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "slipcover" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/slipcover-{{ version }}.tar.gz
+    sha256: 80433ab2ccc9d2cda69f3c5adca4d978c34f13efb551c36e32657598b8c207be
+  # missing file from 0.2.0 sdist required for build
+  # fixed in:
+  # https://github.com/plasma-umass/slipcover/pull/31
+  - path: pyptr.h
+  - patches:
+    - 0001-use-env-build-flags.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - setuptools
+  host:
+    - pip
+    - python >=3.8,<3.12
```

As suggested by the linter. No need for skips, these are the current versions currently supported on conda-forge.
```suggestion
    - python
```